backend/test/forgotPassword.test.js

```diff
@@ -0,0 +1,125 @@
+const { expect } = require('chai');
+const chaiHttp = require('chai-http');
+const chai = require('chai');
+const Admin = require('../app/models/Admin');
+const server = require('../index');
+
+chai.use(chaiHttp);
```

Comment:
This will be later refactored. Ideallt test folder should be similar to the code folder structure. You need not do anything as of now

Reply:
I didn't quite get you. Can you elaborate on what you're trying to say.

Reply:
Ideally test folder structure should resonate with the code folder structure. So for this we need to abstract tests into modules such that the server is started only once and after completion of all tests we stop the server

Reply:
I'll leave it as it is now and you later refactor it according to your needs